### DISCUSSION TITLE
Is there a way to remove the mute button on tabs?

### DISCUSSION THREAD

**Discussion by k1logr4m** [zen-browser/desktop#362] [Q&A] [answered]
I always end up muting the tab instead of switching to it. It appears dead center on the tab and I don't want to "dodge" the button every time. I would rather right-click the tab to mute it.

**Comment by IAmJafeth** [comment, marked as answer]
Hi @k1logr4m ,

A new Zen Theme has been created for this purpose, you can find it here and install it with just one click: https://www.zen-browser.app/themes/053a3ffa-9233-4554-88f3-076e6a6ebb43

![image](https://github.com/user-attachments/assets/3f962857-e5da-4142-9bf2-31236a16ccb8)


Let me know if you have any questions

**Comment by talski** [comment]
It's not working, when I mute a tab, this icon appears, at some point I will click this by mistake

**Comment by Kisakay** [comment]
the page is 404

**Comment by Kisakay** [comment]
I found a workarrounds:

do in the \<Profile Directory\>/chrome/userChrome.css 
```css
.tab-audio-button {
  display: none !important;
}

.tab-icon-overlay[muted] {
  display: none !important;
}
```

Setting to true `toolkit.legacyUserProfileCustomizations.stylesheets = true`
reloading the browser and its good (see attachment)
<img width="208" height="48" alt="image" src="https://github.com/user-attachments/assets/d850ee30-d097-4215-a2d9-a0cfd27879f2" />
